pyproject.toml

```diff
@@ -138,7 +138,8 @@ features = ["development"]
 
 [tool.hatch.envs.integration.scripts]
 test = [
-  "pytest -m integration -n6 --dist=worksteal --deflake-test-type=integration --ignore=tests_integration/spcs",
+  "python -m site",
+  "pytest -m integration -n6 --dist=worksteal -s --deflake-test-type=integration --ignore=tests_integration/spcs -k test_nativeapp_bundle_throws_error_on_no_artifacts -k test_nativeapp_feature_counter_has_expected_value",
```

Comment:
guessing you're trying to bypass current failures, but obviously we can't approve this yet